app/views/api_docs/reference/v1.0-pre/_reference.md

```diff
@@ -3066,20 +3084,62 @@ Deletes an existing degree for this trainee.
       </p>
     </dd>
   </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>course_min_age</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        integer
+      </p>
+      <p class="govuk-body">
+        The lower bound of the age range of children taught on the course (read-only).
+      </p>
+      <p class="govuk-body">
+        Example: <code>7</code>
+      </p>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>course_max_age</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        integer
+      </p>
+      <p class="govuk-body">
+        The upper bound of the age range of children taught on the course (read-only).
+      </p>
+      <p class="govuk-body">
+        Example: <code>11</code>
+      </p>
+    </dd>
+  </div>
   <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
     <dt class="govuk-summary-list__key"><code>course_age_range</code></dt>
     <dd class="govuk-summary-list__value">
       <p class="govuk-body">
         string (limited to 5 characters), required
       </p>
       <p class="govuk-body">
-        The age range of the course. Coded according to the <a href="https://www.hesa.ac.uk/collection/c23053/e/ittphsc">HESA ITT phase/scope field</a>
+        The age range of children taught on the course. Coded according to the <a href="https://www.hesa.ac.uk/collection/c23053/e/ittphsc">HESA ITT phase/scope field</a>
       </p>
       <p class="govuk-body">
         Example: <code>13918</code>
       </p>
     </dd>
   </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>lead_partner_urn</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        string
+      </p>
+      <p class="govuk-body">
+        The Unique Reference Number (URN) of the lead partner for the trainee.
+      </p>
+      <p class="govuk-body">
+        Example: <code>123456</code>
+      </p>
+    </dd>
+  </div>
```

Comment:
Is this a duplicate section for `lead_partner_urn` along with [L3185](https://github.com/DFE-Digital/register-trainee-teachers/pull/4644/files#diff-7d2dd919e95c0e1689162aa5e4998ea21e3d36f7e12e6d4214c4597b28d03621R3185)?